24/02/21 (18:43:07) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1327, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
24/02/21 (19:31:11) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1327, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
24/02/21 (19:31:12) | ERROR: Data collection thread failed with exception: integer division or modulo by zero
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 2925, in _runner
    collector._draw()
  File "/usr/bin/bpytop", line 3953, in _draw
    ProcBox._draw_fg()
  File "/usr/bin/bpytop", line 2800, in _draw_fg
    val = ((v if v <= 100 else 100) + 100) - calc * 100 // cls.select_max
ZeroDivisionError: integer division or modulo by zero
24/02/21 (19:31:12) | WARNING: Exiting with errorcode (1). Runtime 0:00:02 

24/02/21 (19:31:18) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1327, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
24/02/21 (19:31:19) | ERROR: Data collection thread failed with exception: integer division or modulo by zero
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 2925, in _runner
    collector._draw()
  File "/usr/bin/bpytop", line 3953, in _draw
    ProcBox._draw_fg()
  File "/usr/bin/bpytop", line 2800, in _draw_fg
    val = ((v if v <= 100 else 100) + 100) - calc * 100 // cls.select_max
ZeroDivisionError: integer division or modulo by zero
24/02/21 (19:31:19) | WARNING: Exiting with errorcode (1). Runtime 0:00:01 

24/02/21 (19:31:24) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1327, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
24/02/21 (19:31:25) | ERROR: Data collection thread failed with exception: integer division or modulo by zero
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 2925, in _runner
    collector._draw()
  File "/usr/bin/bpytop", line 3953, in _draw
    ProcBox._draw_fg()
  File "/usr/bin/bpytop", line 2800, in _draw_fg
    val = ((v if v <= 100 else 100) + 100) - calc * 100 // cls.select_max
ZeroDivisionError: integer division or modulo by zero
24/02/21 (19:31:25) | WARNING: Exiting with errorcode (1). Runtime 0:00:01 

24/02/21 (19:31:32) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1327, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
24/02/21 (19:36:30) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1327, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
01/03/21 (15:18:01) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1327, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
01/03/21 (16:01:49) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1327, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
01/03/21 (16:03:02) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1327, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
01/03/21 (16:05:23) | INFO: Loglevel set to INFO
18/03/21 (13:47:45) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1327, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
13/04/21 (01:03:14) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1327, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
19/04/21 (12:16:46) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1327, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
19/04/21 (13:56:28) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1327, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
14/05/21 (18:19:11) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1335, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
18/05/21 (16:43:59) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1335, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
27/05/21 (14:24:55) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1335, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
01/06/21 (21:25:12) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1335, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
01/06/21 (22:31:39) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1335, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
12/07/21 (18:16:51) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
27/08/21 (22:30:11) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
30/08/21 (15:19:26) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
30/08/21 (15:55:47) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
19/10/21 (17:48:00) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
04/11/21 (16:37:51) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
04/11/21 (16:49:03) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
05/11/21 (19:19:53) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
06/11/21 (12:02:34) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
12/11/21 (02:23:06) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
13/11/21 (22:30:41) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
25/11/21 (23:54:22) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
25/11/21 (23:54:24) | ERROR: Data collection thread failed with exception: Sign not allowed in string format specifier
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 2939, in _runner
    collector._draw()
  File "/usr/bin/bpytop", line 3971, in _draw
    ProcBox._draw_fg()
  File "/usr/bin/bpytop", line 2719, in _draw_fg
    label = (f'{THEME.title}{Fx.b}{Mv.to(y, x)}{"Pid:":>7} {"Program:" if prog_len > 8 else "Prg:":<{prog_len}}' + (f'{"Arguments:":<{arg_len-4}}' if arg_len else "") +
ValueError: Sign not allowed in string format specifier
25/11/21 (23:54:24) | WARNING: Exiting with errorcode (1). Runtime 0:00:02 

25/11/21 (23:54:34) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
25/11/21 (23:54:35) | ERROR: Data collection thread failed with exception: integer division or modulo by zero
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 2939, in _runner
    collector._draw()
  File "/usr/bin/bpytop", line 3971, in _draw
    ProcBox._draw_fg()
  File "/usr/bin/bpytop", line 2814, in _draw_fg
    val = ((v if v <= 100 else 100) + 100) - calc * 100 // cls.select_max
ZeroDivisionError: integer division or modulo by zero
25/11/21 (23:54:36) | WARNING: Exiting with errorcode (1). Runtime 0:00:02 

25/11/21 (23:54:47) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
07/12/21 (12:20:31) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
13/12/21 (02:40:27) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1337, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
11/01/22 (11:10:59) | ERROR: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/josuer08/.config/bpytop/themes'
